finetune.go

```diff
@@ -0,0 +1,226 @@
+package chatgpt
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"strconv"
+)
+
+type FineTuningJobStatus string
+
+const (
+	FINETUNINGJOBSTATUS_CREATED   FineTuningJobStatus = "created"
+	FINETUNINGJOBSTATUS_PENDING   FineTuningJobStatus = "pending"
+	FINETUNINGJOBSTATUS_RUNNING   FineTuningJobStatus = "running"
+	FINETUNINGJOBSTATUS_SUCCEEDED FineTuningJobStatus = "succeeded"
+	FINETUNINGJOBSTATUS_FAILED    FineTuningJobStatus = "failed"
+	FINETUNINGJOBSTATUS_CANCELLED FineTuningJobStatus = "cancelled"
+)
+
+type FineTuningJob struct {
+	Object          string              `json:"object"`
+	Id              string              `json:"id"`
+	Model           string              `json:"model,omitempty"`
+	CreatedAt       int                 `json:"created_at"`
+	FinishedAt      int                 `json:"finished_at,omitempty"`
+	FineTunedModel  string              `json:"fine_tuned_model,omitempty"`
+	OrganizationId  string              `json:"organization_id,omitempty"`
+	ResultFiles     []string            `json:"result_files,omitempty"`
+	Status          FineTuningJobStatus `json:"status,omitempty"`
+	ValidationFile  interface{}         `json:"validation_file,omitempty"`
+	TrainingFile    string              `json:"training_file,omitempty"`
+	Hyperparameters struct {
+		NEpochs int `json:"n_epochs"`
+	} `json:"hyperparameters,omitempty"`
+	TrainedTokens int    `json:"trained_tokens,omitempty"`
+	Level         string `json:"level,omitempty"`
+	Message       string `json:"message,omitempty"`
+	Data          string `json:"data,omitempty"`
+	Type          string `json:"type,omitempty"`
```

Comment:
These are not valid anymore :(

Reply:
@ayush6624 this PR is kinda a year old. I don't have any need for this anymore, therefore you can close it.